frontend/src/main/resources/puf/frisbee/frontend/view/WaitingView.fxml

```diff
@@ -1,21 +1,37 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
 <?import java.net.URL?>
+<?import javafx.geometry.Insets?>
 <?import javafx.scene.control.Button?>
 <?import javafx.scene.image.Image?>
 <?import javafx.scene.image.ImageView?>
 <?import javafx.scene.layout.AnchorPane?>
+<?import javafx.scene.layout.VBox?>
+<?import javafx.scene.shape.Circle?>
 <?import javafx.scene.text.Text?>
 
-<AnchorPane prefHeight="720.0" prefWidth="1280.0" styleClass="root" xmlns="http://javafx.com/javafx/15" xmlns:fx="http://javafx.com/fxml/1" fx:controller="puf.frisbee.frontend.view.WaitingView">
+<AnchorPane prefHeight="720.0" prefWidth="1280.0" styleClass="root" xmlns="http://javafx.com/javafx/17" xmlns:fx="http://javafx.com/fxml/1" fx:controller="puf.frisbee.frontend.view.WaitingView">
```

Comment:
Hier muss wieder java15 rein, oder?

Reply:
`WARNING: Loading FXML document with JavaFX API of version 17 by JavaFX runtime of version 15.0.1`  ... aber es funktioniert. Wenn das noch geändert wird, dann würde ich approven :)